app/angular/package.json

```diff
@@ -33,6 +33,8 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
+    "@angular/compiler-cli": "^8.2.8",
```

Comment:
These two dependencies are required for this package but hoisted from a different location (`examples/angular-cli` [here](https://github.com/storybookjs/storybook/blob/next/examples/angular-cli/package.json#L23) and [here](https://github.com/storybookjs/storybook/blob/next/examples/angular-cli/package.json#L36)).

These two additions to `dependencies` guarantees that they're here in this package. If these are missing, the build and tests will fail since `yarn install` nests them in `example/angular-cli` after changing TypeScript to `~3.6.0` for some reason.

🤷‍♂